node_modules/oae-rest/README.md

```diff
@@ -0,0 +1,21 @@
+The `rest` module for OAE.
```

Comment:
The oae-rest module lives in a [separate repository](https://github.com/oaeproject/oae-rest) from Hilary so changes to it should be submitted there.